extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -177,104 +196,153 @@ export class AmpIframe extends AMP.BaseElement {
 
 
   /** @override */
-  layoutCallback() {
-    if (!this.isClickToPlay_) {
-      this.assertPosition();
+  isRelayoutNeeded() {
```

Comment:
[`#layoutCallback`](https://github.com/ampproject/amphtml/blob/master/src/service/resources-impl.js#L1670) will not be called unless [`#isRelayoutNeeded`](https://github.com/ampproject/amphtml/blob/master/src/service/resources-impl.js#L1656-L1661) returns `true` too.